src/eureka/S5_lightcurve_fitting/models/BatmanModels.py

```diff
@@ -371,10 +382,10 @@ def eval(self, channel=None, **kwargs):
                     continue
 
                 # Compute light travel time
-                if self.compute_ltt:
+                if self.compute_ltt and (c == 0):
```

Comment:
I'm pretty sure this won't work for multwhite fits. I initially had something like this check, but recently removed it because it was causing some annoyances. The other important thing to check is that the orbital parameters should not vary freely between wavelengths if doing a shared fit (which should definitely be the case, but users currently aren't forced to do that). I haven't done a test to see how fast the LTT step is, but it didn't seem worth the hassle

Reply:
Yeah, I had noticed you made the switch and wasn't sure why.  I'm fine with adopting your solution.